controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -90,13 +91,57 @@ public void testEventKey() {
         assertEquals(commitEvent.getKey(), "test/test");
     }
 
+    @Test(timeout = 10000)
+    public void testisReady() {
```

Comment:
nit: Test name should be testIsReady (Camel Case)

Reply:
done